sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/SpanDataMapper.java

```diff
@@ -366,8 +366,14 @@ private static String getTargetOrNullStableSemconv(Attributes attributes, int de
             return peerService;
         }
         String host = attributes.get(SemanticAttributes.SERVER_ADDRESS);
+        if (host == null) {
+            host = attributes.get(SemanticAttributes.NETWORK_PEER_ADDRESS);
+        }
         if (host != null) {
             Long port = attributes.get(SemanticAttributes.SERVER_PORT);
+            if (port == null) {
+                port = attributes.get(SemanticAttributes.NETWORK_PEER_PORT);
+            }
             return getTarget(host, port, defaultPort);
         }
```

Comment:
(to make it clear that they are pairs)
```suggestion
        String host = attributes.get(SemanticAttributes.SERVER_ADDRESS);
        if (host != null) {
            Long port = attributes.get(SemanticAttributes.SERVER_PORT);
            return getTarget(host, port, defaultPort);
        }
        host = attributes.get(SemanticAttributes.NETWORK_PEER_ADDRESS);
        if (host != null) {
            Long port = attributes.get(SemanticAttributes.NETWORK_PEER_PORT);
            return getTarget(host, port, defaultPort);
        }
```